invenio_app_ils/ui/backoffice/src/common/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+// In development create a personal token to be able to perform requests and
+// and put it in a .env file under the name `REACT_APP_JWT_TOKEN`
+const token =
```

Comment:
I am afraid this will fail on production because of the query on dom. I would do something similar to this.

```
// In development create a personal token to be able to perform requests and
// and put it in a .env file under the name `REACT_APP_JWT_TOKEN`

const environment = process.env.NODE_ENV === 'production' ? 'production' : 'development';
const devToken = environment === 'development' ? process.env.REACT_APP_JWT_TOKEN : '';

const axiosIls = axios.create({
  baseURL:
    process.env.NODE_ENV === 'development'
      ? 'https://localhost:5000/api'
      : '/api',
  withCredentials: true,
});

setBearer() {
  let bearer = environment == 'production' ? (document.getElementsByName('authorized_token') || '') : devToken;
  if(bearer.length) {
    bearer = bearer[0].value;
    return {
      headers: {
        Authorization: `bearer ${bearer}`
      }
    }
  }
}
export const fetchRecord = (url, recid) => {
  return axiosIls.get(`${url}/${recid}`, setBearer());
};

export const postRecord = (url, data) => {
  return axiosIls.post(url, data, setBearer());
};
```

Reply:
Also probably we need to have in the common module a constants.js with all the global constants of our app such as the 
```
const environment = process.env.NODE_ENV === 'production' ? 'production' : 'development';
const devToken = environment === 'development' ? process.env.REACT_APP_JWT_TOKEN : '';
```